i18n/en.json

```diff
@@ -14,5 +14,6 @@
 	"wikibasereconcileedit-editendpoint-invalid-entity-input-version": "unknown entity input version",
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
 	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
-	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must be of type value"
+	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must be of type value",
+	"wikibasereconcileedit-editendpoint-matched-multiple-items": "Matched $2 Items during reconciliation :(\n\nThe matched items are: $1\n"
```

Comment:
I feel like this should use `{{PLURAL}}`. In English, it will never select the singular version of the message, but some other languages have other distinctions than “one” / “other”, and including `{{PLURAL}}` in the English source message makes it clearer for translators that they can also use it in translations, I think. But I’m not sure how this is handled in other messages.

Reply:
Makes sense. Done.